src/commands/retention-policies/assign.js

```diff
@@ -42,6 +50,31 @@ RetentionPoliciesAssignCommand.flags = {
 	'assign-to-id': flags.string({
 		description: 'The ID of the content to assign the retention policy to',
 	}),
+	'filter-fields': flags.string({
```

Comment:
Shouldn't we use a singular form here as we can only add one filter per entry?
`filter-fields` => `filter-field`